lms/templates/pdf_viewer.html

```diff
@@ -197,9 +197,9 @@
                   <span data-l10n-id="download_label">Download</span>
                 </button>
                 <!-- <div class="toolbarButtonSpacer"></div> -->
-                <button id="viewBookmark" class="toolbarButton bookmark hiddenSmallView" title="Current view (copy or open in new window)" data-l10n-id="bookmark">
+                <a href="#" id="viewBookmark" class="toolbarButton bookmark hiddenSmallView" title="Current view (copy or open in new window)" data-l10n-id="bookmark">
```

Comment:
This restores the original intent of this toolbar item-- to be able to copy a "deep link" to the page and location within the page for opening in another window. JavaScript updates the href as you move within the PDF (though it is empty at the time the a11y test runs, so the a11y test will remain disabled).